src/api/dbaas/logs/kibana/kibana.aapi.service.js

```diff
@@ -0,0 +1,26 @@
+angular.module('ovh-api-services').service('OvhApiDbaasLogsKibanaAapi', ($resource, $cacheFactory) => {
+    const cache = $cacheFactory('OvhApiDbaasLogsKibanaAapi');
+  
+    const kibana = $resource('/dbaas/logs/:serviceName/kibana/:kibanaId', {
+      serviceName: '@serviceName',
+      kibanaId: '@kibanaId',
+    }, {
+      get: {
+        method: 'GET',
+        serviceType: 'aapi',
+        cache,
+        isArray: false,
+      },
+    });
+  
+    kibana.resetAllCache = function () {
+        kibana.resetCache();
+    };
+  
+    kibana.resetCache = function () {
+        cache.removeAll();
+    };
+  
+    return kibana;
+  });
+  
```

Comment:
Extra empty line (please refer to the EditorConfig file.

Reply:
Fixed.